### DISCUSSION TITLE
Git LFS

### DISCUSSION THREAD

**Discussion by fheigwer** [community/community#22906] [Enterprise]
<p>Dear Community,</p>
<p>I am a Research Scientist at the German Cancer Research Center. We use github for depositing data, that supplement our scientific publications.</p>
<p>Now I uploaded a file that exceeded out LFS quota and the directory is locked.</p>
<p>How can I delete this file and free the quota without having to delete and redo the entire package?</p>
<p>It is very important that the history stays stable until the mistaken file upload such that the continuity of our repo is ensured.</p>
<p>Any help is much appreciated.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/fheigwer">@fheigwer</a>,</p>
<p>Are you using LFS with <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> or does the German Cancer Research Center use GitHub Enterprise?</p>
<p>If you’re using <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>, I would recommend contacting GitHub Support directly via the <a href="https://git.io/c" rel="nofollow noopener">GitHub Support Portal</a> so that they can look at the account directly and can help troubleshoot the quota.</p>
<p>If you’re using GitHub Enterprise, you will need to contact your local GitHub Enterprise administrator.</p>
<p>Thanks!</p>

**Comment by fheigwer** [comment]
<p>Thanks,</p>
<p>It is indeed what eventually made the trick. The support of Git LFS was very responsive.</p>
